apps/dh/app-dh/jest.config.ts

```diff
@@ -14,7 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-module.exports = {
+/* eslint-disable */
```

Comment:
AFAIK it doesn't have to be at the top of the file, in this instance the comment disables eslint for the remainder of the file (it works like an `eslint-disable` block without the following `eslint-enable`). But sure, it is probably there for a reason, just seems a little aggressive to disable all of eslint instead of the specific rules causing trouble. Hopefully it will be removed again 🙂